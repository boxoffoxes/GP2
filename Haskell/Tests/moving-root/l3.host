[
    (n0 (R), empty)
    (n1, empty)
    (n2, empty)
|
    (e0, n0, n1, empty)
    (e1, n1, n2, empty)
]
