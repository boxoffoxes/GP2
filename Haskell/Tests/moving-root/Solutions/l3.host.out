[
 (n1, empty)
 (n2 (R), empty)
 (n3, empty)
|
 (e1, n2, n1, empty)
 (e3, n3, n2, empty)
]
